x-pack/plugins/lens/public/xy_visualization/visualization.tsx

```diff
@@ -111,15 +124,16 @@ export const getXyVisualization = ({
     };
   },
 
-  appendLayer(state, layerId) {
+  appendLayer(state, layerId, layerType) {
     const usedSeriesTypes = uniq(state.layers.map((layer) => layer.seriesType));
     return {
       ...state,
       layers: [
         ...state.layers,
         newLayerState(
           usedSeriesTypes.length === 1 ? usedSeriesTypes[0] : state.preferredSeriesType,
-          layerId
+          layerId,
+          layerType as LayerType
```

Comment:
why is `layerType` typed as string and then here asserted here to `LayerType`? Cannot you just define type in `types.ts` file?